crates/bitwarden-error-macro/Cargo.toml

```diff
@@ -10,7 +10,7 @@ license-file.workspace = true
 keywords.workspace = true
 
 [features]
-wasm = ["bitwarden-error/wasm"]
```

Comment:
-macro does not depend on bitwarden-error. Had to remove it here, and instead add it to dev-dependencies to ensure tests would compile.